core/backend/src/Schema.ts

```diff
@@ -21,6 +22,18 @@ export class Schema {
    */
   public static get schemaName(): string { throw new Error(`you must override static schemaName in ${this.name}`); }
 
+  private static _schemaKey?: SchemaKey;
+
+  /** Unique identifier for this schema, typed variant of [[schemaName]].
+   * @internal
+   */
+  public static get schemaKey(): SchemaKey {
```

Comment:
If this Schema object represents the version of the schema the domain handlers are written against you could make an argument that the version does matter.

I think what you have done is fine for now but our thinking here should be critical of the purpose of the Schema objects.  That may guide us to change this later.

Reply:
I'm ok resolving this comment, but we need some place to track things we want to ask consumers so we do not loose track of what we have to ask